src/styles/code.css

```diff
@@ -0,0 +1,6 @@
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-top: 4px;
```

Comment:
'code' is your block. We can not give external styling like margin or positioning to block. You can create an element and then give it the external styling.
refer checklist number 6